metagraph/integration_tests/test_clean.py

```diff
@@ -207,58 +207,30 @@ def setUp(self):
     @parameterized.expand(['succinct', 'bitmap', 'hash'])  # , 'hashstr']:
     def test_no_cleaning_contigs(self, representation):
 
-        construct_command = '{exe} build --mask-dummy \
-                --graph {repr} -k 31 --mode canonical --count-kmers -o {outfile} {input}'.format(
-            exe=METAGRAPH,
-            repr=representation,
-            outfile=self.tempdir.name + '/graph',
-            input=TEST_DATA_DIR + '/transcripts_1000.fa'
-        )
-
-        res = subprocess.run([construct_command], shell=True, stdout=PIPE)
-        self.assertEqual(res.returncode, 0)
-
-        stats_command = '{exe} stats {graph}'.format(
-            exe=METAGRAPH,
-            graph=self.tempdir.name + '/graph' + graph_file_extension[representation],
-        )
-        res = subprocess.run(stats_command.split(), stdout=PIPE)
-        self.assertEqual(res.returncode, 0)
+        self._build_graph(input=TEST_DATA_DIR + '/transcripts_1000.fa',
+                          output=self.tempdir.name + '/graph',
+                          k=31, repr=representation, canonical=True,
```

Comment:
maybe let's add modes to `_build_graph_` as well

Reply:
done